distributed/comm/tests/test_ucx_config.py

```diff
@@ -48,27 +53,32 @@ async def test_ucx_config(cleanup):
         "cuda_copy": False,
     }
 
-    with dask.config.set(ucx=ucx):
+    with dask.config.set({"distributed.comm.ucx": ucx}):
         ucx_config = _scrub_ucx_config()
-        assert ucx_config.get("TLS") == "rc,tcp,sockcm"
-        assert ucx_config.get("SOCKADDR_TLS_PRIORITY") == "sockcm"
+        if ucx_110:
+            assert ucx_config.get("TLS") == "rc,tcp"
+            assert ucx_config.get("SOCKADDR_TLS_PRIORITY") == "tcp"
+        else:
+            assert ucx_config.get("TLS") == "rc,tcp,sockcm"
+            assert ucx_config.get("SOCKADDR_TLS_PRIORITY") == "sockcm"
         assert ucx_config.get("NET_DEVICES") == "mlx5_0:1"
 
     ucx = {
         "nvlink": False,
         "infiniband": True,
         "rdmacm": True,
         "net-devices": "all",
-        "MEMTYPE_CACHE": "y",
```

Comment:
Just curious, why are we dropping this?

Reply:
We don't support passing UCX variables via Dask config anymore, before you could do `DASK_UCX__MEMTYPE_CACHE=y`, but there's really no reason for that when a user can just use UCX variables directly such as `UCX_MEMTYPE_CACHE=y`.

Reply:
Great, thanks for the clarification